applications/GeoMechanicsApplication/python_scripts/geomechanics_T_solver.py

```diff
@@ -0,0 +1,170 @@
+import KratosMultiphysics
+import KratosMultiphysics.GeoMechanicsApplication as KratosGeo
+from KratosMultiphysics.GeoMechanicsApplication.geomechanics_solver import GeoMechanicalSolver as GeoSolver
+
+
+def CreateSolver(model, custom_settings):
+    return TSolver(model, custom_settings)
+
+
+class TSolver(GeoSolver):
+    """Solver for the solution of thermal problems."""
+
+    def __init__(self, model, custom_settings):
+        super().__init__(model, custom_settings)
+
+        KratosMultiphysics.Logger.PrintInfo("GeoMechanics_T_Solver", "Construction of Solver finished.")
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        this_defaults = KratosMultiphysics.Parameters("""{
+            "solver_type": "geomechanics_T_solver",
+            "model_part_name": "PorousDomain",
+            "domain_size": 2,
+            "start_time": 0.0,
+            "model_import_settings":{
+                "input_type": "mdpa",
+                "input_filename": "unknown_name"
+            },
+            "material_import_settings" :{
+                "materials_filename": ""
+            },
+            "time_stepping": {
+                "end_time" : 1.0,
```

Comment:
For the other solvers, `"end_time"` was moved to `"problem_data"` rather than `"solver_settings"`. Can we do the same thing here?

Reply:
It is removed here but it is still in geomechanics for cosim